react/features/help/components/HelpView.js

```diff
@@ -0,0 +1,119 @@
+// @flow
+
+import React, { PureComponent } from 'react';
+import { SafeAreaView, View } from 'react-native';
+
+import { HeaderWithNavigation, SlidingView } from '../../base/react';
+import { ColorSchemeRegistry } from '../../base/color-scheme';
+import { translate } from '../../base/i18n';
+import { connect } from '../../base/redux';
+import { StyleType } from '../../base/styles';
+
+import { setHelpViewVisible } from '../actions';
+import WebView from 'react-native-webview';
+
+import styles from './styles';
+
+type Props = {
+
+    /**
+     * The color schemed style of application headers.
+     */
+    _headerStyles: StyleType,
+
+    /**
+     * The color schemed style of feature.
+     */
+    _styles: StyleType,
+
+    /**
+     * True if the view should be rendered, false otherwise.
+     */
+    _visible: boolean,
+
+    /**
+     * The Redux Dispatch function.
+     */
+    dispatch: Function
+}
+
+/**
+ * Implements a page that renders the help content for the app.
+ */
+class HelpView extends PureComponent<Props> {
+    /**
+     * Instantiates a new component.
+     *
+     * @inheritdoc
+     */
+    constructor(props: Props) {
+        super(props);
+
+        this._onRequestClose = this._onRequestClose.bind(this);
+    }
+
+    /**
+     * Implements {@code PureComponent#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+    render() {
+        const { _headerStyles, _styles } = this.props;
+
+        return (
+            <SlidingView
+                onHide = { this._onRequestClose }
+                position = 'bottom'
+                show = { this.props._visible } >
+                <View
+                    style = { [
+                        _headerStyles.page,
+                        _styles.page
+                    ] }>
+                    <HeaderWithNavigation
+                        headerLabelKey = 'helpView.header'
+                        onPressBack = { this._onRequestClose } />
+                    <SafeAreaView style = { styles.safeArea }>
+                        <WebView source = {{ uri: 'https://web-cdn.jitsi.net/faq/meet-faq.html' }} />
```

Comment:
Let's make this configurable please. Configurable as in config.js, that is, with that as a default.